src/axom/core/CMakeLists.txt

```diff
@@ -100,10 +96,11 @@ set(core_sources
 #------------------------------------------------------------------------------
 
 set( core_depends fmt )
+blt_list_append( TO core_depends ELEMENTS adiak::adiak IF ADIAK_FOUND )
+blt_list_append( TO core_depends ELEMENTS caliper IF CALIPER_FOUND )
 blt_list_append( TO core_depends ELEMENTS camp IF CAMP_FOUND )
 blt_list_append( TO core_depends ELEMENTS umpire IF UMPIRE_FOUND )
 blt_list_append( TO core_depends ELEMENTS RAJA IF RAJA_FOUND )
-blt_list_append( TO core_depends ELEMENTS nvToolsExt IF AXOM_ENABLE_CUDA )
```

Comment:
I confirmed that after removing this, the `nvtx` caliper service still works .
I.e. it adds the proper annotations w/ nvidia's `nsys` and `nsys-ui`  (I'll post screenshots and command lines).